src/app/dspace/components/full-item-view.component.ts

```diff
@@ -43,6 +43,12 @@ import {Item} from '../models/item.model'
 })
 export class FullItemViewComponent {
 
+    /**
+     * An object that represents the current item.
+     *
+     * TODO: replace any with inheritance model. e.g. item extends dspaceObject
+     */
+    itemJSON: any;
```

Comment:
Yes, it was breaking the breadcrumb without it. The other pull request maps all the responses to models and no longer uses json in code.